it_zwierzak/adoption_announcement/views.py

```diff
@@ -25,21 +27,25 @@ def adoption_form(request):
     return render(request, 'adoption_announcement/adoption_application.html')
 
 
-class AdoptionApplicationView(LoginRequiredMixin, FormView): #tylko dla zalogowanych
+def confirmation_adoption_application(request):
+    return render(request, 'confirmation_adoption_application.html')
+
+
+class AdoptionApplicationView(LoginRequiredMixin, FormView):
```

Comment:
to raczej nie jest wykorzystanie FormView zgodnie z przeznaczeniem, zmieńmy na View